### PR TITLE
MOBN-1348 | Differentiate the Released versions of a Dictionary from the Unreleased ones

### PR DESCRIPTION
**Pull request by vinishad**
…nary from unreleased version

# JIRA TICKET NAME:
[MOBN-1348](https://msfprojects.atlassian.net/browse/MOBN-1348)

# Summary:
Differentiate the Released versions of a Dictionary from the Unreleased ones

### CODE REVIEW

File: src/apps/dictionaries/components/__test__/ReleasedVersions.test.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import '@testing-library/jest-dom'
+import {
+    BrowserRouter as Router,
+} from "react-router-dom";
+
+type releasedVersionProps = React.ComponentProps<typeof ReleasedVersions>;
+
+const baseProps: releasedVersionProps = {
+    versions: [],
+    subscriptionUrl: "Subscription URL",
```
Comment (kirity-mahanty) [inline src/apps/dictionaries/components/__test__/ReleasedVersions.test.tsx]: subscriptionUrl and linkedSource are removed from interface Props defined for ReleasedVersions.
We should remove these from the test cases

File: src/apps/dictionaries/components/ReleasedVersions.tsx [hunk 2]
```diff
@@ -89,6 +90,16 @@ const ReleasedVersions: React.FC<Props> = ({
                         </Tooltip>
                       </CopyToClipboard>
                     </TableCell>
+                      <TableCell>
+                              <Switch
+                                  id={row.id}
+                                  data-testid={row.id}
+                                  checked={row.released}
+                                   // onChange={handleChange}
+                                  name="checkedB"
```
Comment (kirity-mahanty) [inline src/apps/dictionaries/components/ReleasedVersions.tsx]: May be rename it to `checkedReleaseStatus`

**Review by kirity-mahanty — COMMENTED** [review]
Also change the pipeline so that it runs the unit tests.

File: src/apps/dictionaries/components/__test__/ReleasedVersions.test.tsx (hunk 1 above)
Comment (vinishad) [reply src/apps/dictionaries/components/__test__/ReleasedVersions.test.tsx]: Done @kirity-mahanty

File: src/apps/dictionaries/components/ReleasedVersions.tsx (hunk 2 above)
Comment (vinishad) [reply src/apps/dictionaries/components/ReleasedVersions.tsx]: Done

**Pull request merged**